node/rpc-client/src/chain_ops.rs

```diff
@@ -2,32 +2,32 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use super::client::filecoin_rpc;
-use blocks::{header::json::BlockHeaderJson, tipset_json::TipsetJson};
 use cid::{json::CidJson, Cid};
+use rpc_api::chain_api::*;
+
 use jsonrpc_v2::Error as JsonRpcError;
-use message::unsigned_message::json::UnsignedMessageJson;
 
 /// Returns a block with specified CID fom chain via RPC
-pub async fn block(cid: Cid) -> Result<BlockHeaderJson, JsonRpcError> {
-    filecoin_rpc::chain_get_block(CidJson(cid)).await
+pub async fn block(cid: Cid) -> Result<ChainGetBlockResult, JsonRpcError> {
```

Comment:
Refactor:
`Result<ChainGetBlockResult, JsonRpcError>` -> `ChainGetBlockResult`, and elsewhere:

```rust
pub type RPCResult<T> = Result<T, JsonRpcError>;
pub async fn block(cid: Cid) -> RPCResult<ChainGetBlockResult> {
    ...
}
```